src/sentry/integrations/jira/utils/group.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+from functools import reduce
+from typing import Any, Mapping
+from urllib.parse import quote
+
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.group import StreamGroupSerializer
+from sentry.models import Group
+from sentry.utils.http import absolute_uri
+
+
+def accum(tot: int, item: Mapping[int, int]) -> int:
+    return tot + item[1]
+
+
+# TODO: find more efficient way of getting stats
+def get_serialized_and_stats(group: Group, stats_period: str) -> tuple[Mapping[str, Any], int]:
```

Comment:
Should we leave serialization/deserialization in the views? Keep request related logic there while moving business logic in utils?